script/unicode/Unicode/UnicodeSkipList.lean

```diff
@@ -0,0 +1,148 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+import Init.Data.Char.UnicodeSkipList
+import Unicode.Unicode
+import Unicode.Parse
+import Init.Data.Char.Basic
+
+open Std System IO FilePath FS
+
+namespace Char.UnicodeSkipList
+
+/-
+The following code creates the unicode skip list data structure for
+a given property. To understand it at a high-level, we provide an
+explanation in Init/Data/Char/UnicodeSkipList.lean
+-/
+
+/-
+Break down the sequence of codepoints into a sequence of ranges,
+alternating between ranges that satisfy the property and ranges that
+do not.
```

Comment:
I don't think this is true? To me the code looks like it only returns the ranges that satisfy the property?

Reply:
I confirm that both kind of ranges are represented (e.g. for the numeric table, the first range of 48 values or any range marked as 0). It is crucial since determining if a codepoint has the property is based on the parity of the index of the range.

Reply:
I updated the comments. That property is true, but only later.